scripts/gen_quick_start_module.py

```diff
@@ -142,12 +149,11 @@ def update_versions(versions, release_matrix, release_version):
                                 for ver in [RELEASE, DEBUG]:
                                      instr["versions"][LIBTORCH_DWNL_INSTR[ver]] = rel_entry_dict[ver]
                         elif os_key == OperatingSystem.MACOS.value:
-                            rel_entry_dict = {
-                                x["devtoolset"]: x["installation"] for x in pkg_arch_matrix
-                                if x["libtorch_variant"] == "shared-with-deps"
-                                }
                             if instr["versions"] is not None:
-                                instr["versions"][LIBTORCH_DWNL_INSTR[MACOS]] = list(rel_entry_dict.values())[0]
+                                instr["versions"][LIBTORCH_DWNL_INSTR[MACOS]] = pkg_arch_matrix[0]["installation"]
+                                # todo: remove this once we release libtorch for arm64 as stable
+                                if release_version == "nightly":
+                                    instr["versions"][LIBTORCH_DWNL_INSTR[MACOS_ARM64]] = pkg_arch_matrix[1]["installation"]
```

Comment:
This feel wrong, it should add those instructions of more flavors are available for the platform, rather than hardcode  MacOS or anything like that here

Reply:
Removed nightly check